CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## [Unreleased]
 
+### Added
+
+-
+
+### Fixed
+
+- `get_mlflow_config` now uses the kedro context config_loader to get configs (#66 #64 #54 #30 #31 #72)
```

Comment:
Could you change this line to be more explicit about the changes? It ill help users to find out the very important breakthrough you've made, and eventually to look in CHANGELOG later if needed (with a ctrl+f) .Something like (but feel free to change the wording): 

```markdown
- `get_mlflow_config` now uses the kedro context config_loader to get configs (#66). This indirectly solves the following issues:
    -  `get_mlflow_config` now works in interactive mode if `load_context` is called  with a path different from the working directory (#30)
    - kedro_mlflow now works fine with kedro jupyter notebook independently of the working directory (#64) 
    - You can use global variables in `mlflow.yml` which  is now properly parsed if you use a `TemplatedConfigLoader`  (#72)
This change does not fix, but paves the way to further enhancement:
    - Add the possibility to manage environment variables through kedro `credentials.yml` file (#31) 
```

Reply:
yes, it's more informative ! that show concretely what user gain from the changes.
But the last one `- Add the possibility to manage environment variables through kedro credentials.yml file` can be a bit confusing, as it's not fixed yet. Pointing further enhancement enabled (or facilitated) by a change will complexify the changelog readability.
I suggest to adress the last point when it is fixed. probably with resolving this issue #77

Reply:
Agreed. We'll make a comment in #77 after this PR is merged to keep track that it is now resolvable.